api/v1/ytsaurus_types.go

```diff
@@ -457,6 +457,9 @@ type UISpec struct {
 	Theme       string  `json:"theme,omitempty"`
 	Description *string `json:"description,omitempty"`
 	Group       *string `json:"group,omitempty"`
+	// When this is set to true, UI will use backend for downloading instead of proxy. If this is set to false or omitted, UI use proxies, which is a default behaviour.
+	//+optional
+	DisableDirectDownload bool `json:"disableDirectDownload,omitempty"`
```

Comment:
Usually, in golang code I'm in favor of not using pointers if nil doesn't have special meaning for us. 
But our practice shows that in operator crd specs it is better to use pointers. I suggest we better revert to `*bool`, because if we use non-pointer `bool` `kubectl apply` will show diff `disableDirectDownload: false` for users who haven't changed anything in spec and don't expect diff. Writing down `false` default value in the applied spec is also not a very good idea, because it makes our future possible default change decisions problematic.